package.json

```diff
@@ -5,6 +5,7 @@
   "main": "lib/index",
   "scripts": {
     "test": "npm run lint && mocha",
+    "test:all": "nve 8 10 12 \"*\" mocha",
```

Comment:
I'd rather not do this just yet. My main concern is the current assumption is that the user has `nve` installed globally; which I'm not comfortable with. And I'm not sure whether it'd mean to bring this in as a dev dependency. Let's discuss this in a separate PR if you're passionate about it.

Reply:
I think you might be confusing `nve` with `nvm`. nve is installed locally and can be used to run tests against different node version *without* installing them globally.
[Read](https://github.com/ehmicky/nve/blob/master/README.md) about it

Reply:
I can confirm `nve` is installed locally, not globally (I created that package).
If you need any support or have any questions, please let me know directly or on the GitHub repo :)